src/Plugin.php

```diff
@@ -39,7 +39,11 @@ public static function init() {
    */
   public static function renderCustomMessage() {
     $source_code_message = get_option('source_code_message');
-    echo '<!-- ' . $source_code_message . ' -->';
+    // Prevent from displaying message in sitemap files.
+    $url = $_SERVER['REQUEST_URI'];
+    if (strpos($url,'.xml') === false) {
```

Comment:
This is far too late in the request processing to make a differentiation. Instead, we don't want to register the filter for feed responses in the first place.

Reply:
Yep, totally agree — sorry to not noticed this before.